.idea/workspace.xml

```diff
@@ -2,7 +2,11 @@
 <project version="4">
```

Comment:
What's this file for? Is it maybe committed by mistake?

Reply:
I dont know. I dont update it just gets updated itself. I even ignored it in .gitignore.

Reply:
Shouldn't it be `.idea/`?
I think you also have to remove the folder, commit, like they said here: https://stackoverflow.com/questions/32384473/gitignore-not-ignoring-idea-path 

Or just merge this and I can try to do it separately